lib/js/components/SelectList/SelectListItemToggle/SelectListItemToggle.stories.ts

```diff
@@ -0,0 +1,82 @@
+import SelectListItemToggle from './SelectListItemToggle.vue';
+import { ICONS } from '../../Icons/Icon';
+
+import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue';
+import { SELECT_LIST_SIZES } from '../SelectList.consts';
+
+export default {
+	title: 'Components/SelectList/SelectListItemToggle',
+	component: SelectListItemToggle,
+} as Meta<typeof SelectListItemToggle>;
+
+const StoryTemplate: StoryFn<typeof SelectListItemToggle> = (argTypes, { updateArgs }) => ({
+	components: { SelectListItemToggle },
+	props: Object.keys(argTypes),
+	template: `
+		<select-list-item-toggle
+			:icon-off="ICONS[$props.iconOff]"
+			:icon-on="ICONS[$props.iconOn]"
```

Comment:
why do you use `$props` in stories?

Reply:
Copy paste from Button story. I'll clean it up.